google-cloud-pubsub/src/test/java/com/google/cloud/pubsub/it/ITPubSubTest.java

```diff
@@ -16,127 +16,129 @@
+import com.google.pubsub.v1.PushConfig;
+import com.google.pubsub.v1.SubscriptionName;
+import com.google.pubsub.v1.TopicName;
+import java.util.Collections;
 import java.util.List;
 import java.util.UUID;
-import java.util.concurrent.ExecutionException;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.Timeout;
 
-public class ITPubSubTest extends BaseSystemTest {
+public class ITPubSubTest {
 
-  private static final PubSub PUB_SUB = PubSubOptions.getDefaultInstance().getService();
   private static final String NAME_SUFFIX = UUID.randomUUID().toString();
+  private static PublisherClient publisherClient;
+  private static SubscriberClient subscriberClient;
+  private static String projectId;
 
-  @Rule
-  public Timeout globalTimeout = Timeout.seconds(300);
+  @Rule public Timeout globalTimeout = Timeout.seconds(300);
 
-  @Override
-  protected PubSub pubsub() {
-    return PUB_SUB;
+  @BeforeClass
+  public static void setupClass() throws Exception {
+    publisherClient = PublisherClient.create();
+    subscriberClient = SubscriberClient.create();
+    projectId = ServiceOptions.getDefaultProjectId();
   }
 
-  @Override
-  protected String formatForTest(String resourceName) {
-    return resourceName + "-" + NAME_SUFFIX;
+  @AfterClass
+  public static void tearDownClass() throws Exception {
+    publisherClient.close();
+    subscriberClient.close();
   }
 
-  // Policy tests are defined here and not in BaseSystemTest because Pub/Sub emulator does not
-  // support IAM yet
+  private String formatForTest(String resourceName) {
+    return resourceName + "-" + NAME_SUFFIX;
+  }
 
   @Test
   public void testTopicPolicy() {
-    String topicName = formatForTest("test-topic-policy");
-    Topic topic = pubsub().create(TopicInfo.of(topicName));
-    Policy policy = pubsub().getTopicPolicy(topicName);
-    policy = pubsub().replaceTopicPolicy(topicName, policy.toBuilder()
-        .addIdentity(Role.viewer(), Identity.allAuthenticatedUsers())
-        .build());
-    assertTrue(policy.getBindings().containsKey(Role.viewer()));
-    assertTrue(policy.getBindings().get(Role.viewer()).contains(Identity.allAuthenticatedUsers()));
-    List<Boolean> permissions =
-        pubsub().testTopicPermissions(topicName, ImmutableList.of("pubsub.topics.get"));
-    assertTrue(permissions.get(0));
-    topic.delete();
-  }
+    TopicName topicName = TopicName.create(projectId, formatForTest("testing-topic-policy"));
+    publisherClient.createTopic(topicName);
 
-  @Test
-  public void testNonExistingTopicPolicy() {
-    String topicName = formatForTest("test-non-existing-topic-policy");
-    assertNull(pubsub().getTopicPolicy(topicName));
-  }
+    Policy policy = publisherClient.getIamPolicy(topicName.toString());
+    Binding binding =
+        Binding.newBuilder().setRole("roles/viewer").addMembers("allAuthenticatedUsers").build();
```

Comment:
It's a shame we're losing the typed IAM classes (Role, Identity, Policy) - not something you can address in this PR, though.